jib-gradle-plugin/src/test/java/com/google/cloud/tools/jib/gradle/JibPluginTest.java

```diff
@@ -370,6 +373,37 @@ public void testJibTaskGroupIsSet() {
         taskName -> Assert.assertEquals(taskName, "Jib", tasks.getByPath(taskName).getGroup()));
   }
 
+  @Test
+  public void testLazyEvalForImageAndTags() {
+    try {
+      testProject.build(JibPlugin.BUILD_IMAGE_TASK_NAME);
+      Assert.fail("Expect this to fail");
+    } catch (UnexpectedBuildFailure ex) {
+      String output = ex.getBuildResult().getOutput().trim();
```

Comment:
Does it makes sense to remove this whole regex and just do

cleanOutput = output.replace("color-stuff","")

Assert.assertThat(cleanOutput, Matchers.contains("Containerizing application....")

Reply:
Nice. Although, `assertThat` seems to have been deprecated so I used `assertTrue(matcher.find())` instead?

Reply:
Oh yeah, you can use `MatcherAssert` like in: https://github.com/GoogleContainerTools/jib/blob/cc930f213a66c3895e5e7f3890566a3f50acce97/jib-maven-plugin/src/test/java/com/google/cloud/tools/jib/maven/skaffold/InitMojoTest.java#L66